develop/reference/articles/xx-theme-file-reference/00-theme-reference-intro.markdown

```diff
@@ -0,0 +1,241 @@
+# Theme Reference Guide [](id=theme-reference-guide)
+
+A theme is made up of several files. Although most of the files are named after 
+their matching component, you may be unsure of the function of some of these 
+files.
+
+This document explains each file's usage in more detail, so that you will know
+which files to modify and which files to leave untouched.
+
+## Theme Anatomy [](id=theme-anatomy)
+
+There are two main approaches to theme development for @product@ 7: themes built 
+using the Node build tools with the [themes generator](/develop/tutorials/-/knowledge_base/7-0/themes-generator) 
+and themes built using the Plugins SDK.
+
+For themes developed with the themes generator, your theme anatomy is organized
+with the structure shown below. Although themes developed with the Plugins SDK 
+have a slightly different anatomy, the core theme files are the same. Note that 
+the `build` directory is shown for reference, and is not created by default.
```

Comment:
Maybe instead of saying it's not created by default, just say it's generated when the theme is compiled? I don't want new theme devs thinking it's something that have to manually create.